docs/pages/tv-settings_halSpec.md

```diff
@@ -0,0 +1,514 @@
+Picture profile database will have 5 types of tables:
+1. Picture property table to maintain all picture properties for a given picture mode, video format and video source
+2. Picture mode association table maintain the association of a given video source and video format to a picture mode
+3. White balance table for every color temperature and video source to maintain WB calibrated values
+4. Gamma table for every color temperature to maintain the gamma calibrated values.
+5. TMAX table for every local dimming level to maintain the TMAX value
+
+- The specification of the TV picure configuration will be defined in a config file ([pq_capablities.ini](../../config/pq_capablities.ini "pq_capablities.ini")) which decides supported formats, picture modes, dimming modes, dvModes, resolution etc.
+- Caller must initialize by calling `tvInit()` which must initialize the parameters in picture property table. These parameters are decided by Soc vendor  based on platform capability.
+- On every bootup the default picture profile database will be a copied of override picture profile database.  
+
+#### Table Format
+##### Picture Property Table
+| Video Source | Video Format | Picture Mode | Picture Property |
+| --- | --- | --- | --- |
+| HDMI1 | SDR | Standard/Entertainment | Brightness |
+|  |  |  | Contrast |
+|  |  |  | Sharpness |
+|  |  |  | Saturation |
+|  |  |  | Hue |
+|  |  |  | Backlight |
+|  |  |  | Dolbymode |
+|  |  |  | AspectRatio |
+|  |  |  | Colortemperature |
+|  |  |  | Dimming Mode |
+|  |  |  | Local Dimming |
+|  |  |  | Low Latency |
+|  |  |  | CMS State |
+|  |  |  | CMS Saturation RED |
+|  |  |  | CMS Saturation BLUE |
+|  |  |  | CMS Saturation GREEN |
+|  |  |  | CMS Saturation YELLOW |
+|  |  |  | CMS Saturation CYAN |
+|  |  |  | CMS Saturation MAGENTA |
+|  |  |  | CMS Hue RED |
+|  |  |  | CMS Hue BLUE |
+|  |  |  | CMS Hue GREEN |
+|  |  |  | CMS Hue YELLOW |
+|  |  |  | CMS Hue CYAN |
+|  |  |  | CMS Hue MAGENTA |
+|  |  |  | CMS Luma RED |
+|  |  |  | CMS Luma BLUE |
+|  |  |  | CMS Luma GREEN |
+|  |  |  | CMS Luma YELLOW |
+|  |  |  | CMS Luma CYAN |
+|  |  |  | CMS Luma MAGENTA |
+|  |  | Theater/Movie | Repeat as standard |
+|  |  | FilmMaker | Repeat as standard |
+|  |  | Sports | Repeat as standard |
+|  |  | Game | Repeat as standard |
+|  |  | Custom/Expert | Repeat as standard |
+|  |  | EnergySaving | Repeat as standard |
+|  |  | Vivid | Repeat as standard |
+|  |  | Graphics | Repeat as standard |
+|  | HDR10 | Repeat as SDR |
+|  | HLG | Repeat as SDR |
+|  | DV | Repeat as SDR |
+| HDMI2 | Repeat as HDMI1 |
+| HDMI3 | Repeat as HDMI1 |
+| Tunner | Repeat as HDMI1 |
+| IP | Repeat as HDMI1 |
+| Composite | Repeat as HDMI1 for SDR |

```

Comment:
Do we have different picture modes on composite?  I think we discussed this and though of setting to SDR and Entertainment picture mode.  if that is the case, can we mention the same here? same in the picture association table as well.

Reply:
Done